app/helpers/appointment_helper.rb

```diff
@@ -2,4 +2,24 @@ module AppointmentHelper
   def friendly_options(statuses)
     statuses.map { |k, _| [k.titleize, k] }.to_h
   end
+
+  def rebooked_from(appointment)
+    return unless appointment.rebooked_from_id?
+
+    content_tag(
+      :span,
+      "rebooked from ##{appointment.rebooked_from_id}",
```

Comment:
I slipped a change in quick so maybe this is moot now but I didn't think it made sense linking back as I couldn't imagine a flow where they'd want to go back to the original appointment they're booking from, especially since the original's details are copied here anyway.

Happy to alter though if anyone feels strongly about it.